code/modules/ventcrawl/ventcrawl.dm

```diff
@@ -0,0 +1,223 @@
+var/list/ventcrawl_machinery = list(
+	/obj/machinery/atmospherics/unary/vent_pump,
+	/obj/machinery/atmospherics/unary/vent_scrubber
+	)
+
+// Vent crawling whitelisted items, whoo
+/mob/living/var/list/can_enter_vent_with = list(
+	/obj/item/weapon/implant,
+	/obj/item/device/radio/borg,
+	/obj/item/weapon/holder,
+	/obj/machinery/camera,
+	/mob/living/simple_animal/borer
+	)
+
+/mob/living/var/list/icon/pipes_shown = list()
+/mob/living/var/last_played_vent
+/mob/living/var/is_ventcrawling = 0
+/mob/var/next_play_vent = 0
+
+/mob/living/proc/can_ventcrawl()
+	return 0
+
+/mob/living/Login()
```

Comment:
I wonder if this should be taken back to the original definition of mob/living/Login. Unless the proc is already parsed out?